Packages/io.chainsafe.web3-unity.loader/Editor/ScopedRegistryInstaller.cs

```diff
@@ -0,0 +1,125 @@
+#if UNITY_EDITOR
+using UnityEditor;
+using UnityEngine;
+using System.IO;
+using System.Linq;
+using System.Text;
+using Newtonsoft.Json.Linq;
+
+[InitializeOnLoad]
+public static class ScopedRegistryAndDependencyInstaller
+{
+    private static readonly string RegistryName = "package.openupm.com";
+    private static readonly string RegistryUrl = "https://package.openupm.com";
+    private static readonly string[] RequiredScopes =
+    {
+        "com.reown.appkit.unity",
```

Comment:
I think you can just add com.reown as a required scope and it'll be able to add anything from that "namespace" as a dependency

Reply:
I'm like 95% sure xD